crates/ruff_linter/resources/test/fixtures/pyflakes/F821_26.py

```diff
@@ -42,3 +42,7 @@ class MyClass:
 baz: MyClass
 eggs = baz  # valid in a `.pyi` stub file, not in a `.py` runtime file
 eggs = "baz"  # always okay
+
+if TYPE_CHECKING:
```

Comment:
Can we a test for if there is code in an else block after the TYPE_CHECKING? That shouldn't be treated a stub, right?

Reply:
Sure, I can add a test for that.